metron-analytics/metron-profiler-client/src/main/java/org/apache/metron/profiler/client/HBaseProfilerClient.java

```diff
@@ -44,27 +44,30 @@
   /**
    * Used to access the profile data stored in HBase.
    */
-  private HTableInterface table;
+  private HBaseClient hbaseClient;
```

Comment:
The `HBaseProfilerClient` was updated to use our `HBaseClient` abstraction to interact with HBase.  This also allows us to use the `FakeHBaseClient` when testing this class.